napalm/junos/junos.py

```diff
@@ -325,7 +325,43 @@ def get_facts(self):
             "interface_list": interface_list,
         }
 
+    def get_vlans(self):
+        result = {}
+        vlans = junos_views.junos_vlan_table(self.device)
+        vlans.get()
+
+        # VLAN information comes in a tuple so like interfaces
+        # we convert to dict
+        def _convert_to_dict(vlans):
+                vlans = dict(vlans)
+                
+                for vlan, vlan_data in vlans.items():
+
+                        result[vlan] = { 
+                                "name": vlan_data["name"],
+                                "tag": ( 
+                                        True
+                                        if vlan_data["tag"] is None
+                                        else vlan_data["tag"]
+
+                                ),
+                                "members": (
+                                        "none-assigned" # Checking even older junos, it alwayas returns either a list of members or l2ng-l2rtb that you see below
```

Comment:
This should be just an empty list instead of "none-assigned".

Reply:
OK I can change that. I put this as a catch-all but an empty list as a catch all works too

Reply:
How's it going? :)

Reply:
I have been working on implementing this along side network-importer tools (https://github.com/networktocode/network-importer/) but I can submit the change you recommended to get this moved up. I originally started this because of of network-importer so I wanted to make sure it'd work with it

Reply:
As you wish, @ckozler - if you want to continue working on this with the requested changes, and the tests, I'll be happy to review; otherwise, feel free to close it out. :+1: